logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/TeamEventReceiver.kt

```diff
@@ -33,12 +41,35 @@ class TeamEventReceiverImpl(
         )
             .onFailure { kaliumLogger.e("$TAG - failure on member join event: $it") }
 
-    private suspend fun handleMemberLeave(event: Event.Team.MemberLeave) =
+    private suspend fun handleMemberLeave(event: Event.Team.MemberLeave) {
+        val userId = UserId(event.memberId, selfUserId.domain)
         teamRepository.removeTeamMember(
             teamId = event.teamId,
             userId = event.memberId,
-        ).onSuccess { conversationRepository.deleteUserFromConversations(UserId(event.memberId, selfUserId.domain)) }
+        )
+            .onSuccess {
```

Comment:
I think we have to update the tests for this new logic ? 🤔

Reply:
Yup